PopulationLib/Dispersal/Distance2Parent/Distance2Parent.py

```diff
@@ -26,23 +26,26 @@ def getInputParameters(self, args):
 
         # Set default values if not defined in project file
         if not hasattr(self, "scale"):
-            self.scale = 1
-            print("INFO: Default value for <dispersal><FixedRate><scale> is used. Default: 1")
+            if hasattr(self, "distribution") and self.distribution == "weibull":
```

Comment:
```suggestion
            if self.distribution == "weibull":
```
-> `distribution` is a required tag

Reply:
erledigt